tests/regression_tests/ncrystal/test.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import openmc
+import openmc.lib
+import pytest
+
+from tests.testing_harness import PyAPITestHarness
+
+pytestmark = pytest.mark.skipif(
+    not openmc.lib._ncrystal_enabled(),
+    reason="NCrystal materials are not enabled.")
+
+def compute_angular_distribution(cfg, E0, N):
```

Comment:
A name like `pencil_beam_model` might better convey the fact that this returns a Model object

Reply:
This was done now.